classes/navigationMenu/NavigationMenuItem.inc.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/**
+ * @file classes/navigationMenu/NavigationMenuItem.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class NavigationMenuItem
+ * @ingroup navigationMenu
+ * @see NavigationMenuItemDAO
+ *
+ * @brief Basic class describing a NavigationMenuItem.
+ */
+
+/** ID codes and paths for all default navigationMenuItems */
+define('NMI_ID_CURRENT',	0x00000001);
```

Comment:
I'm not sure, but I think these might essentially be the symbolic names we were recommending. Are these supposed to be generated in parallel by the AUTOINCREMENT column so that they correspond? If so, that's an iffy assumption -- we've had trouble e.g. mixing AUTOINCREMENT and directly-assigned values before; PostgreSQL's sequence structure along with ADODB's database maintenance stuff and whatever else happens on a PostgreSQL-based server mean that I suspect we'll have systems that don't correspond to our expectations. I'd suggest using a `symbolic` column with `STRING_CONSTANTS` to identify the built-in menus. That way we can hopefully avoid having to sync a registry file with constants defined here, which will likely also get forgotten.